homework2.js

```diff
@@ -0,0 +1,95 @@
+function task3() {
+    var a = parseInt(+prompt("Введите число"));
+    var b = parseInt(+prompt("Введите число"));
+    if (a >= 0 && b >= 0) {
+        var result = a - b;
+    } else if (a < 0 && b < 0) {
+        var result = a * b;
+    } else if ((a < 0 && b > 0) || (a > 0 && b < 0)) {
```

Comment:
Эта дополнительная проверка не нужна
```suggestion
    } else {
```